frontend/e2e/pages/metrics/config-step.ts

```diff
@@ -156,6 +160,10 @@ export class ConfigStep {
     this.sourceControlTokenErrorMessage = this.sourceControlContainer.getByText('Token is incorrect!');
 
     this.saveAsButton = page.getByRole('button', { name: 'Save' });
+    this.addNewTimeRangeButton = page.getByRole('button', { name: 'Button for adding date range' });
+    this.errorStartTimeMessage = page.getByText('Start date is invalid');
+    this.errorEndTimeMessage = page.getByText('End date is invalid');
+    this.removeTimeRangeButtons = page.getByText('Remove');
```

Comment:
Let's move these locators right behind `requiedMetrics***` ones, since previously we put the locators by the kinds of `basicInfo` `pipelineTool` and `sourceControl`.

Reply:
done